src/Desktop/UIAutomation/EventHandlers/TextEditTextChangedEventListener.cs

```diff
@@ -35,7 +35,7 @@ public override void Init()
             }
         }
 
-        public void HandleTextEditTextChangedEvent(IUIAutomationElement sender, TextEditChangeType type, string[] array)
+        public void HandleTextEditTextChangedEvent(IUIAutomationElement sender, TextEditChangeType TextEditChangeType, string[] eventStrings)
```

Comment:
Another instance of the auto-generated interface using inconsistent casing of parameters, that perhaps we want to suppress

Reply:
[Same case here](https://docs.microsoft.com/en-us/windows/win32/api/uiautomationclient/nf-uiautomationclient-iuiautomationtextedittextchangedeventhandler-handletextedittextchangedevent), will suppress!

Reply:
Done.